fastlane/Fastfile

```diff
@@ -673,10 +673,11 @@ end
 def check_no_git_tag_exists(version_number)
   if git_tag_exists(tag: version_number, remote: true, remote_name: 'origin')
     raise "git tag with version #{version_number} already exists!"
+  else
+    UI.message("git tag with version #{version_number} doesn't exist")
```

Comment:
let's maybe add something like ✅  to the message to make it clearer that it's a good thing, since the message from the command looks ominous

Reply:
alternatively we could replace the fastlane command with a direct git command so there's no shell error code

Reply:
hmm I've been trying the alternative, and it looks like Fastlane's Action.sh will always log the non-zero status code. For example, I tried this command

```
Actions.sh("git ls-remote --tags origin #{version_number} | grep -q #{version_number}")
```

Which returns 1 if the tag doesn't exist and 0 if the tag exists and fastlane was still logging 

```
Exit status of command 'git ls-remote --tags origin 4.18.0-SNAPSHOT | grep -q 4.18.0-SNAPSHOT' was 1 instead of 0.
```

Maybe there's a way of preventing the log from showing, but I think I will just go with the checkmark for now ✅